engine/execution/ingestion/engine.go

```diff
@@ -1372,9 +1374,27 @@ func (e *Engine) fetchCollection(blockID flow.Identifier, height uint64, guarant
 		)
 		return fmt.Errorf("could not find guarantors: %w", err)
 	}
+
+	filters := []flow.IdentityFilter{
+		filter.HasNodeID(guarantors...),
+	}
+
+	// This is included to temporarily work around an issue observed on a small number of ENs.
+	// Using only the onflow LNs seems to aleviate the issue. This will be removed once a
+	// proper fix is in place.
+	if os.Getenv("ONFLOW_LN_ONLY_OVERRIDE") != "" {
```

Comment:
do we want to use flag instead of env var?

Reply:
the reason I used an envvar was so operators wouldn't accidentally discover and use this flag. Also, the cli flags are somewhat of an API for operators, so we should avoid adding and removing flags unnecessarily. Undocumented envvars make no promises

Reply:
We have prefix the flag with `temp_`, and add comments to suggest this is a temporary flag.

I think it's easier to manage with flags than evnvar, as EN prints all the flags that we have configured on startup. So that if we notice any behavior we will be able to check what was the actual flags being used. 

With `envvar`, we would also need to print the env var as well